drivers/volume/portworx/portworx.go

```diff
@@ -770,6 +770,16 @@ func (d *portworx) DeleteVolume(volumeID string) error {
 	return nil
 }
 
+func (d *portworx) CreateSnapshot(volumeID string, snapName string) (*api.SdkVolumeSnapshotCreateResponse, error) {
+	volDriver := d.getVolDriver()
+	snapshotResponse, err := volDriver.SnapshotCreate(d.getContext(), &api.SdkVolumeSnapshotCreateRequest{VolumeId: volumeID, Name: snapName})
+	if err != nil {
+		return nil, fmt.Errorf("Error while creating snapshot on the volume %v due to %v", volumeID, err)
```

Comment:
nit: Add snapshot name you are creating, good for debugging purposes 
```
return nil, fmt.Errorf("Error while creating snapshot [%s] on the volume [%s], Err: %v", snapName, volumeID, err)
```

Reply:
I was in an impression that err will show the complete response wrt snapname and snapID  when any failure occurs 
so I intentionally skipped providing snapName in the error out, anyways I have updated it now